### PR TITLE
Add Dockerfile & docker-compose.yml for BetyDB

### PR DESCRIPTION
**Pull request by max-zilla**
To use, cd to the bety directory with docker installed and simply say:
```
docker-compose up
```

This will create a container with a fresh PostGIS install and then create a clean BetyDB container to connect to it. Once the compose script has completed you should see something like this:
```
bety_1     | [2016-03-15 19:36:44] INFO  WEBrick 1.3.1
bety_1     | [2016-03-15 19:36:44] INFO  ruby 2.1.8 (2015-12-16) [x86_64-linux]
bety_1     | [2016-03-15 19:36:44] INFO  WEBrick::HTTPServer#start: pid=104 port=3000
```

This means Bety is ready. In a new terminal use 
```
docker ps -a
```
...to see your running docker containers. There will be a port number listed in the line with bety, e.g.
```
0.0.0.0:32789->3000/tcp
```

You can access bety at 192.168.99.100/port locally, e.g. in this example

```
192.168.99.100/32789
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# ruby & bety
+bety:
+  image: maxzilla2/betydb
```
Comment (robkooper) [inline docker-compose.yml]: can you make this pecan/bety

**Comment by robkooper** [comment]
Did you intend to commit script/buildSQLGeom.py and modify all permissions on the files.

**Comment by robkooper** [comment]
should not remove Gemfile.lock

File: docker/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    # Create bety database
+    createdb -U postgres -h $PG_PORT_5432_TCP_ADDR -p $PG_PORT_5432_TCP_PORT bety
+
+    # Download & initialize Bety database contents
+    cd ../script
+    curl -LOs https://raw.githubusercontent.com/PecanProject/pecan/master/scripts/load.bety.sh
+    chmod +x load.bety.sh
+    ./load.bety.sh -a "postgres" -p "-h $PG_PORT_5432_TCP_ADDR -p $PG_PORT_5432_TCP_PORT" -o postgres -c -u -g -r 0 -m 99
+
```
Comment (robkooper) [inline docker/entrypoint.sh]: Should do a rake db::migrate before running

**Comment by max-zilla** [comment]
Closing this pull request - will open another shortly after getting git house in order.

**Comment by max-zilla** [comment]
New pull request: https://github.com/PecanProject/bety/pull/395/files

**Pull request closed without merging**